runatlantis.io/docs/policy-checking.md

```diff
@@ -120,3 +120,37 @@ deny[msg] {
 ```
 
 That's it! Now your Atlantis instance is configured to run policies on your Terraform plans 🎉
+
+## Customizing the conftest command
+
+### Pulling policies from a remote location
+
+Conftest supports [pulling policies](https://www.conftest.dev/sharing/#pulling) from remote locations such as S3, git, OCI, and other protocols supported by the [go-getter](https://github.com/hashicorp/go-getter) library. You can use [`extra_args`](https://www.runatlantis.io/docs/custom-workflows.html#adding-extra-arguments-to-terraform-commands) and pass in the [`--update`](https://www.conftest.dev/sharing/#-update-flag) flag to tell conftest to pull the policies into the project folder before running the policy check.
+
+```
+workflows:
+  custom:
+    plan:
+      steps:
+        - init
+        - plan
+    policy_check:
+      steps:
+        - policy_check:
+            extra_args: ["--update", "s3::https://s3.amazonaws.com/bucket/foo"]
+```
+
+Note that you may need to configure authentication separately if you want to pull policies from sources that requires it. For example, to pull policies from an S3 bucket, you can configure your Atlantis host with a default AWS profile that has permission to read from the S3 bucket.
+
+### Running policy check against Terraform source code
+
+By default, Atlantis runs the policy check against the [SHOWFILE](https://www.runatlantis.io/docs/custom-workflows.html#custom-run-command) but if you want to run the policy test against Terraform files directly, you can override the default conftest command used and pass in `*.tf` as one of the inputs to conftest. The `show` step is required so that Atlantis will generate the SHOWFILE.
```

Comment:
```suggestion
By default, Atlantis runs the policy check against the [`SHOWFILE`](https://www.runatlantis.io/docs/custom-workflows.html#custom-run-command). In order to run the policy test against Terraform files directly, override the default `conftest` command used and pass in `*.tf` as one of the inputs to `conftest`. The `show` step is required so that Atlantis will generate the `SHOWFILE`.
```